buildSrc/src/software/aws/toolkits/gradle/SourcesUtils.kt

```diff
@@ -12,9 +14,9 @@ import java.io.FileFilter
  * [type] is the type of the source folder (e.g. 'src', 'tst', 'resources')
  * [ideVersion] is the 3 digit numerical version of the JetBrains SDK (e.g. 192, 201 etc)
  */
-fun findFolders(project: Project, type: String, ideVersion: String): List<String> = project.projectDir.listFiles(FileFilter {
+fun findFolders(project: Project, type: String, ideVersion: String): MutableSet<File> = project.projectDir.listFiles(FileFilter {
     it.isDirectory && includeFolder(type, ideVersion, it.name)
-})?.map { it.name } ?: emptyList()
+})?.map { File(it.name) }?.toMutableSet() ?: mutableSetOf()
```

Comment:
why does this need to be mutable?

Reply:
I was doing something wrong in the conversion and it complained about them being immutable, they can be immutable again.